lua/fzf-lua/_extensions/metals.lua

```diff
@@ -0,0 +1,45 @@
+local log = require("metals.log")
+local commands_table = require("metals.commands").commands_table
+
+local has_fzf, fzf = pcall(require, "fzf-lua")
+
+if not has_fzf then
+  local msg = "fzf-lua must be installed to use this functionality (https://github.com/ibhagwan/fzf-lua)"
+  log.error_and_show(msg)
+end
+
+local function run_entry(entry)
+  local command = require("metals")[entry.id]
+  local success, msg = pcall(command)
+  if not success then
+    vim.api.nvim_notify(msg, 2, {})
+  end
+end
+
+local function run_selected(label)
+  for _, entry in ipairs(commands_table) do
+    if entry.label == label then
+      run_entry(entry)
+      return
+    end
+  end
+end
+
+return function(opts)
+  opts = opts or {}
```

Comment:
I don't see one looking through the docs, but I wish there was a way to also include the description that would be shown but not considered when you search. Either way, maybe an improvement for later on if someone knows how to add that.

Reply:
The only way I see is through a preview, as the fzf-lua menu entries are simple strings.